packages/shared/src/components/tooltips/RecommendedEmojiTooltip.tsx

```diff
@@ -0,0 +1,73 @@
+import type { MutableRefObject, ReactElement } from 'react';
+import React from 'react';
+import classNames from 'classnames';
+import * as TooltipPrimitive from '@radix-ui/react-tooltip';
+import { Typography, TypographyTag } from '../typography/Typography';
+
+const RecommendedEmojiTooltip = ({
+  elementRef,
+  search,
+  emojiData = [],
+  onSelect,
+  selected = 0,
+  offset = [0, 0],
+  onClickOutside,
+}: {
+  elementRef: MutableRefObject<HTMLElement>;
+  offset?: number[];
+  search?: string;
+  emojiData?: Array<{ name: string; emoji: string }>;
+  selected?: number;
+  onSelect: (emoji: string) => void;
+  onClickOutside?: () => void;
+}): ReactElement => {
+  if (!search || !emojiData?.length) {
+    return null;
+  }
+
+  const [offsetX, offsetY] = offset;
+
+  return (
+    <TooltipPrimitive.Provider>
```

Comment:
We have Popover too, which might work better than tooltip. I used it in KeywordSelector

Reply:
Agree with this notion :)